spec/lib/mcb/commands/apiv1/providers/list_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'mcb_helper'
+
+describe '"mcb apiv1 providers list"' do
+  let(:current_cycle) { find_or_create(:recruitment_cycle, year: '2019') }
+  let(:next_cycle) { find_or_create(:recruitment_cycle, year: '2020') }
+  let(:provider1) { create(:provider, recruitment_cycle: current_cycle) }
+  let(:provider2) { create(:provider, recruitment_cycle: next_cycle) }
+
+  it 'lists providers' do
+    url = 'http://localhost:3001/api/v1/providers'
```

Comment:
I'm not sure this test makes sense. For API V1 we don't expect to ever get a request without a recruitment cycle, and is documented to only work with the recruitment cycle:

https://github.com/DFE-Digital/manage-courses-backend/blob/master/docs/api.md

I would expect there to be a test to show that if the `-r` option is not provider, only providers from the current cycle are displayed.